src/dashboard/apps/auth/admin.py

```diff
@@ -40,6 +41,38 @@ class DashboardUserAdmin(UserAdmin):
         "is_attached_to_entity",
     )
     list_filter = ("is_staff", "is_superuser", "is_active", "groups", "is_validated")
+    actions = ["validate_users"]
+
+    def save_model(self, request, obj, form, change):
+        """Overriding save_model to handle errors during the save action."""
+        try:
+            super().save_model(request, obj, form, change)
+        except Exception as e:
+            self.message_user(
+                request,
+                _(f"Error during update: {str(e)}"),
+                level=messages.ERROR,
+            )
```

Comment:
This is only targeting single-user update right?

Reply:
Yes.
The goal here is simply to display error messages in a clear way for the user